logrotate/CMakeLists.txt

```diff
@@ -43,20 +60,12 @@ SET(LIBRARY_TO_BUILD g3logrotate)
 
 # Build the library
 INCLUDE_DIRECTORIES(${PROJECT_SRC})
-ADD_LIBRARY(${LIBRARY_TO_BUILD} SHARED ${SRC_FILES})
```

Comment:
If you need the .a library then please duplicerar the libraries so it generates a .so and a .a.